### DISCUSSION TITLE
7 is not recognized

### DISCUSSION THREAD

**Discussion by lindi200000** [jomjol/AI-on-the-edge-device#3464] [General]
### Files

Hello, the 7 is not recognized for me, or very rarely. It is always regarded as a 3.
![main_dig1](https://github.com/user-attachments/assets/13567cde-4dc7-4d69-930e-c0ad76ee5ad0)
![image](https://github.com/user-attachments/assets/9ea7380f-9f51-473b-9d4a-5f10fc3bdade)
![image](https://github.com/user-attachments/assets/8bda59c6-2165-436d-9057-e86154a04ddb)

Does the 7 need to be learned more, or can I somehow improve my recognition?

**Comment by SybexX** [comment]
https://jomjol.github.io/AI-on-the-edge-device-docs/ROI-Configuration/
